### DISCUSSION TITLE
Whats the number count in the input line?

### DISCUSSION THREAD

**Discussion by GronbergInfo** [ohmybash/oh-my-bash#642] [Q&A] [answered]
Eveyr now and then my prompt goes from
`>` to `1>` or `127>`

**Comment by akinomyoga** [comment, marked as answer]
It depends on the theme (`OSH_THEME`) that you chose in your `~/.bashrc`, but it is usually the exit status of the last command you run in the shell. The value 1 stands for the general error. The value 2 usually stands for the usage error. The value 127 usually means that the command was not found.

**Comment by GronbergInfo** [comment]
ah ok. I use the brainy theme. that explains the numbers a bit
